src/index.html

```diff
@@ -3,12 +3,53 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="searchBar">
+      <nav class="first">
+        <ul class="link">
+          <li>
+            <a class="is-active" href="http://APPLE.html">APPLE</a>
+          </li>
+
+          <li>
+            <a href="http://SAMSUNG.html">SAMSUNG</a>
```

Comment:
```suggestion
            <a href="#samsung">Samsung</a>
```
please fix all links